### PR TITLE
Feature/fix active accounts

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Documentation
image

Fees
image

### CODE REVIEW

File: backend/lib/utils.js [hunk 1]
```diff
@@ -182,7 +182,8 @@ module.exports = {
     const { method } = extrinsic.toHuman().method;
     const args = JSON.stringify(extrinsic.args);
     const hash = extrinsic.hash.toHex();
-    const doc = extrinsic.meta.docs.toString().replace(/'/g, "''");
+    // const doc = extrinsic.meta.docs.toString().replace(/'/g, "''");
+    const doc = '';
```
Comment (AndreiNavoichyk) [inline backend/lib/utils.js:188]: Why hardcode?

File: backend/lib/utils.js [hunk 2]
```diff
@@ -220,17 +221,26 @@ module.exports = {
...
+    //       .catch((error) => logger.debug(loggerOptions, `API Error: ${error}`)) || '',
+    //     api.rpc.payment.queryFeeDetails(extrinsic.toHex(), blockHash)
+    //       .then((result) => JSON.stringify(result.toJSON()))
+    //       .catch((error) => logger.debug(loggerOptions, `API Error: ${error}`)) || '',
+    //   ]);
+    // }
+
     let feeInfo = '';
-    let feeDetails = '';
+    const feeDetails = '';
```
Comment (AndreiNavoichyk) [inline backend/lib/utils.js:194]: Why hardcode?

File: backend/lib/utils.js [hunk 3]
```diff
@@ -220,17 +221,26 @@ module.exports = {
     //     .catch(() => {}) || '';
     // }
 
+    // let feeInfo = '';
```
Comment (AndreiNavoichyk) [inline backend/lib/utils.js]: We never commit commented code.

File: backend/lib/utils.js (hunk 1 above)
Comment (karthikeyan-cere) [reply backend/lib/utils.js:188]: Meta doc is not available in the extrinsic. So I have assigned an empty string to it.

File: backend/lib/utils.js (hunk 2 above)
Comment (karthikeyan-cere) [reply backend/lib/utils.js:194]: We don't have the `api.rpc.payment.queryFeeDetails` RPC call, so I have hard coded it to an empty string.

File: backend/lib/utils.js (hunk 3 above)
Comment (karthikeyan-cere) [reply backend/lib/utils.js]: Once we update our pos-network-node with the latest substrate, we can enable this piece of code.

Comment (AndreiNavoichyk) [reply backend/lib/utils.js]: Better to add ToDo with humanreadable reason. Code we can always find in git.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**